client/a8c-for-agencies/components/pressable-usage-details/style.scss

```diff
@@ -86,4 +86,91 @@
 		@include heading-large;
 		margin-top: 5px;
 	}
+
+	/* Titan Email Addon styles */
+	.pressable-usage-details__addon-tag {
+		@include body-small;
+		background-color: var(--color-neutral-5);
+		color: var(--color-neutral-70);
+        font-size: 11px;
+		padding: 2px 6px;
+		border-radius: 4px;
+		margin-left: 6px;
+		text-transform: uppercase;
+	}
+
+	.pressable-usage-details__titan-domains {
+		display: flex;
+		flex-direction: column;
+		gap: 12px;
+		margin-top: 16px;
+        font-size: 14px;
+        font-weight: 600;
+	}
+
+	.pressable-usage-details__titan-domain-item {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		padding: 8px 0;
+		border-bottom: 1px solid var(--color-neutral-5);
+
+		&:last-child {
+			border-bottom: none;
+		}
+	}
+
+	.pressable-usage-details__titan-domain-info {
+		display: flex;
+		align-items: center;
+		flex-wrap: wrap;
+		gap: 8px;
+	}
+
+	.pressable-usage-details__titan-domain {
+		@include body-medium;
+		font-weight: 500;
+	}
+
+	.pressable-usage-details__titan-plan-label {
+		@include body-small;
+		background-color: var(--color-primary-5);
+		color: var(--color-primary-40);
+        font-weight: 500;
+		padding: 2px 6px;
+        margin-left: 6px;
+		border-radius: 4px;
+	}
+
+	.pressable-usage-details__titan-trial-badge {
+		@include body-small;
+		background-color: var(--color-warning-5);
+		color: var(--color-warning-40);
+		padding: 2px 6px;
+		border-radius: 4px;
+		margin-right: 6px;
+	}
+
+	.pressable-usage-details__titan-trial-text {
+		@include body-small;
+		color: var(--color-neutral-60);
+	}
+
+	.pressable-usage-details__titan-inboxes {
+		@include body-small;
+		color: var(--color-neutral-60);
+	}
+
+    .pressable-titan-usage-details__info-top-right {
+        @include body-small;
+        font-weight: 500;
+        font-size: 13px;
```

Comment:
I'm not sure why we're overriding this here. Can't we just include `heading-small` instead of the ' body-small ' if the `body-small` is not sufficient?